app/exporters/formatters/countryside_stewardship_grant_indexable_formatter.rb

```diff
@@ -7,7 +7,12 @@ def type
 
   private
   def extra_attributes
-    {}
+    {
+      grant_type: entity.grant_type,
+      land_use: entity.land_use,
+      tiers_or_standalone_items: entity.tiers_or_standalone_items,
+      funding_per_unit_per_year: entity.funding_per_unit_per_year,
```

Comment:
Could we call this something a bit less of a mouthful, like `funding_amount`? The per unit per year thing feels like a display concern, so something to just put into the label strings.